Example/ViewController.swift

```diff
@@ -603,6 +604,7 @@ class ViewController: UIViewController {
                                                                                    animations: {
                 navigationViewController.navigationView.wayNameView.alpha = 1.0
                 navigationViewController.navigationView.floatingStackView.alpha = 1.0
+                navigationViewController.navigationView.speedLimitView.alpha = 1.0
```

Comment:
Just improved overall animation when presenting and dismissing `NavigationViewController`.

Reply:
Is this compatible with SpeedLimitView’s built-in behavior to hide when there’s no content and to blink when the speed limit updates? The blinking operates on the view’s layer’s opacity.

https://github.com/mapbox/mapbox-navigation-ios/blob/793a34f2f24c9e4eaec2f94079b245cc717b20f2/Sources/MapboxNavigation/SpeedLimitView.swift#L114-L121